docs/src/guides/drupal9/solr.md

```diff
@@ -37,13 +37,15 @@ Once [this issue](https://www.drupal.org/project/drupal/issues/2876675) is resol
 
 If you are using the latest version of Search API Solr and running Drupal 8.8 or higher on PHP 7.2 or higher, this step is no longer necessary. You should be able to proceed directly to the next step.
```

Comment:
> If you are using the latest version of Search API Solr and running Drupal 8.8 or higher on PHP 7.2 or higher, this step is no longer necessary. You should be able to proceed directly to the next step.

This should probably be the first line under:
> ### 0. Upgrade Symfony Event Dispatcher

Reply:
I didn't go through the flow of the page and I think there would be other things to do if we started with that. I was focused only on the process for adding it.